src/App.jsx

```diff
@@ -5,79 +5,146 @@ import Header from "./components/Header";
     constructor() {
         super();
-        this.state = {
-            initializing: true,
-            error: false,
-        };
+        this.state = {loading: false, error: false};
+    }
 
-        /**
-         * Initialize as Kirk and optionally do away team creation if necessary
-         */
+    /**
+     * Trigger initialization and set the loading state to true until initialization completes.
+     */
+    handleInitClick() {
+        this.initIronWebSDK();
+        this.setState({loading: true});
+    }
+
+    /**
+     * Initialize as Kirk and optionally do away team creation if necessary
+     */
+    initIronWebSDK() {
         initializeAsUser(KIRK)
             .then(() => getTestGroupDetails())
             .then((group) => {
                 this.props.setAwayTeam(group);
-                this.setState({initializing: false, error: false});
+                this.setState({loading: false, error: false});
             })
-            .catch(() => this.setState({initializing: false, error: true}));
+            .catch(() => this.setState({loading: false, error: true}));
     }
 
     /**
      * Get markup for when the app is initializing the app.
      */
-    getInitializingMarkup() {
+    getInitPageMarkup() {
+        return (
+            <div className={classes.main}>
+                <Paper height="600px" width="320px">
+                    <div className={classes.initPanel}>
+                        <HorizontalLogoBlack height="50" width="200" alt="IronCore Logo" className={classes.logo} />
+                        <h1 className={classes.mainText}>Initialize IronWeb SDK to get started.</h1>
+                        <Button classes={classes.button} buttonAction={() => this.handleInitClick()}>
+                            Initialize IronWeb SDK
+                        </Button>
+                    </div>
+                </Paper>
+            </div>
+        );
+    }
+
+    /**
+     * Get markup for when the app is loading.
+     */
+    getLoadingMarkup() {
         return (
-            <div className={classes.loaderWrapper}>
+            <div className={classes.main}>
                 <div id="initLoader">
-                    <img src={logo} height="200" width="200" alt="" />
+                    <img src={logo} height="70" width="70" alt="" />
                 </div>
             </div>
         );
     }
 
+    /**
+     * Get markup for when an error initializing the IronWeb SDK occurs.
+     */
+    getErrorMarkup() {
+        return (
+            <div className={classes.main}>
+                <Paper height="600px" width="320px">
+                    <div className={classes.initPanel}>
+                        <h1>Woops! An Error Occured</h1>
+                        <h4 className={classes.mainText}>An error occured while initializing the IronCore service. Please refresh your page to try again.</h4>
+                        <Button buttonAction={window.location.reload()}>
```

Comment:
Don't you just want to pass in `window.location.reload` or maybe `() => window.location.reload()`. Pretty sure you don't want to call it here.